azure-pipelines.yml

```diff
@@ -80,25 +80,15 @@ strategy:
       image: 'ubuntu-20.04'
       cmd: "mvn -e --no-transfer-progress verify"
 
-    # MacOS JDK8 verify
-    'MacOS JDK8 verify':
-      image: 'macOS-10.15'
-      cmd: "export JAVA_HOME=$JAVA_HOME_8_X64 && mvn -e --no-transfer-progress verify"
-
     # MacOS JDK11 verify
     'MacOS JDK11 verify':
       image: 'macOS-10.15'
-      cmd: "export JAVA_HOME=$JAVA_HOME_11_X64 && mvn -e --no-transfer-progress verify"
-
-    # MacOS JDK13 verify
-    'MacOS JDK13 verify':
-      image: 'macOS-10.15'
-      cmd: "export JAVA_HOME=$JAVA_HOME_13_X64 && mvn -e --no-transfer-progress verify"
+      cmd: "JAVA_HOME=$JAVA_HOME_11_X64 mvn -e --no-transfer-progress verify"
```

Comment:
`export JAVA_HOME=$JAVA_HOME_11_X64` does not work with `eval "$(cmd)"`
Replaced as one command with environment variable override.